configapi/api_slice_mgmt.go

```diff
@@ -64,6 +64,10 @@ func DeviceGroupPostHandler(c *gin.Context, msgOp int) bool {
 	if groupName, exists = c.Params.Get("group-name"); exists {
 		logger.ConfigLog.Infof("received group %v", groupName)
 	}
```

Comment:
```suggestion
	groupName := c.Param("group-name")
```
Since `ValidateName` already checks if the string is empty. Could we get rid of this `if` ?
Applies for NS, gNBs and UPF